src/NServiceBus.Core/Timeout/Hosting/Windows/ManageMessageFailuresWithoutSlr.cs

```diff
@@ -45,7 +47,7 @@ void SendFailureMessage(TransportMessage message, Exception e, string reason)
                 return;
             }
 
-            message.SetExceptionHeaders(e, localAddress ?? config.LocalAddress,reason);
+            message.SetExceptionHeaders(e, failedQueue ?? localAddress ?? config.LocalAddress,reason);
```

Comment:
@timbussmann correctly or not, changing the way it is initialized would be a breaking change so is out of scope. I agree this is weird.